lib/alaveteli_spam_term_checker.rb

```diff
@@ -0,0 +1,83 @@
+# -*- encoding : utf-8 -*-
+class AlaveteliSpamTermChecker
+  DEFAULT_SPAM_TERMS = [
+    /Freedom of Information request - [\{\[\|]/i,
+    /Sea?,?s?o?n?[ -.]\d+[ -]Epi?'?s?,?o?d?e?s?[ -.]\d+/i,
+    /\[Full-Watch\]/i,
+    /free-watch/i,
+    /Online Full 2016/i,
+    /HD | Special ".*?" Online/i,
+    /FULL .O?N?_?L?I?N?E? ?\.?MOVIE/i,
+    /\[Full 20x5\]/i,
+    /Putlocker/i,
+    /se?\d+ep?\d+/i,
+    /\[free\]/i,
+    /vodlocker/i,
+    /online free full/i,
+    /streaming 2016/i,
+    /films?-?hd/i,
+    /full online/i,
```

Comment:
Not sure about this as a default – could easily mention "a full online transcript" or something like that.